mittab/libs/outround_tab_logic/checks.py

```diff
@@ -0,0 +1,126 @@
+from mittab.apps.tab.models import *
+from mittab.libs.errors import PrevRoundNotEnteredError
+
+
+def lost_teams():
+    return [outround.loser.id
+            for outround in Outround.objects.all()
+            if outround.loser]
+
+
+def have_enough_judges_type(type_of_round):
+    lost_ids = [outround.loser.id
+                for outround in Outround.objects.all()
+                if outround.loser]
+
+    panel_size = 3
+
+    if type_of_round == BreakingTeam.VARSITY:
+        panel_size = TabSettings.get("var_panel_size", 3)
+    else:
+        panel_size = TabSettings.get("nov_panel_size", 3)
+
+    teams_count = BreakingTeam.objects.filter(
+        type_of_team=type_of_round
+    ).exclude(team__id__in=lost_ids).count()
+
+    judges_needed = (
+        teams_count // 2
+    ) * panel_size
+
+    var_to_nov = TabSettings.get("var_to_nov", 2)
+
+    num_teams = teams_count
+
+    other_round_num = num_teams / var_to_nov
+    if type_of_round == BreakingTeam.NOVICE:
+        other_round_num = num_teams * var_to_nov
+
+    other_round_type = not type_of_round
```

Comment:
Same comment as the view file w/r/t the clarity of this line

Reply:
Done